mkdocs.yml

```diff
@@ -83,3 +84,7 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
+        - name: python
+          class: python
+          validator: !!python/name:markdown_exec.validator
+          format: !!python/name:markdown_exec.formatter
```

Comment:
You don't need to declare the fence if you added `markdown-exec` to the plugin list. The plugin basically declares a fence for each of its supported language.